tests/test_authz_authorizer.py

```diff
@@ -38,37 +32,161 @@ def test__init__(self):
         assert self.authz.resource == "test_resource"
         assert self.authz.permission == "permission_name"
 
-    def test__repr__(self):
+    def test__repr__(self) -> None:
         assert repr(self.authz) == (
             "Authorizer(auth_jwt=<jwt>, "
             "resource_name='test_resource', "
             "permission_name='permission_name')"
         )
 
-    def test__set_policy_w_scope(self):
+    def test_validate_one_with_expired(self) -> None:
+        expiration_negative = int((datetime.utcnow() - timedelta(seconds=1)).timestamp())
+        with pytest.raises(Unauthorized):
+            self._make_authorizer(
+                {
+                    **self.token_payload,
+                    "allow": {"test_resource": {"permission_name": {"allow": ALL}}},
+                    "exp": expiration_negative,
+                }
+            )
+
+
+# pylint: disable=too-many-public-methods
+class TestAuthorizerSetupMethod:
+    @staticmethod
+    def _mocked_make_authorizer(token_payload: dict) -> Authorizer:
+        with patch("lbz.authz.authorizer.decode_jwt", lambda _: token_payload):
+            return Authorizer("xx", "test_resource", "permission_name")
+
+    @pytest.fixture(autouse=True)
```

Comment:
ditto: wrong usage of `pytest` fixtures

Reply:
actually its correct - in this place it enforcing using fixutres as this is not test_ method.

Reply:
Yes, it works as you want, but that is only a side effect. In reality, you have made a fixture based on this method - that is something against the pattern of how the `pytest` fixtures should be used. Please, use fixtures as they should be used, so everywhere directly.